src/Microsoft.Extensions.CodeGeneration.Core/project.json

```diff
@@ -9,6 +9,7 @@
     "Microsoft.Extensions.CodeGeneration.Templating": "1.0.0-*",
     "Microsoft.Extensions.CommandLineUtils": "1.0.0-*",
     "Microsoft.Extensions.DependencyInjection": "1.0.0-*",
+    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*",
```

Comment:
Oh I see, it's transitive from `Microsoft.Extensions.CodeGeneration.Templating`, I'll remove this one then.